services/workflows-service/package.json

```diff
@@ -15,20 +15,19 @@
     "start:watch": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "build": "nest build --path=tsconfig.build.json",
-    "test": "jest --verbose",
+    "test": "jest",
     "test:unit": "cross-env SKIP_DB_SETUP_TEARDOWN=true jest --testRegex '.*\\.unit\\.test\\.ts$'",
     "test:integration": "jest --testRegex '.*\\.intg\\.test\\.ts$'",
     "test:e2e": "jest --testRegex '.*\\.e2e\\.test\\.ts$'",
     "test:watch": "jest --verbose --watch",
     "seed": "ts-node scripts/seed.ts",
     "seed:ex": "ts-node scripts/seed-ex.ts",
-    "db:migrate-save": "prisma migrate dev",
+    "db:migrate-dev": "prisma migrate dev",
```

Comment:
Good change! Worth notifying the team it has been made. 🙏🏼

Reply:
no one actually use it

Reply:
it usually used when running db:init
```
"db:init": "npm run db:migrate-dev -- --name 'initial version' && npm run db:migrate-up seed",
```

Reply:
It is used when making a change to the Prisma schema.